src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1253,6 +1253,7 @@ nns_edge_create_handle (const char *id, nns_edge_connect_type_e connect_type,
   eh->listening = false;
   eh->sending = false;
   eh->listener_fd = -1;
+  eh->caps_str = nns_edge_strdup ("nnsEdgeCapsNotSet");
```

Comment:
Q: init null, why do you set the meaningless string?

Reply:
When caps is not set, an error occurs and the edge process is terminated.
This is to change the determination of the caps by the event recipient.

Reply:
Suggestion: nns_edge_strdup (""); plz check empty string case.

Reply:
Thanks. Updated.